### PR TITLE
Add current and expected json files for save_problem_success event

### PR DESCRIPTION
**Pull request by mumarkhan999**
**Trello Link:** [Click here](https://trello.com/c/fhMha3eE/16-problem-interaction-event-saveproblemsuccess)

**Description:** Contains current and expected json files for save_problem_success

**Checks before merge:**

- [x] JSON Reviewed
- [x]   Code Reviewed
- [x]   Unit Test Pass
- [x]   Commits squashed

### CODE REVIEW

**Comment by asamolion** [comment]
Please begin work on the code

**Comment by mumarkhan999** [comment]
@asamolion waiting for your approval.

File: openedx/features/caliper_tracking/transformers/problem_transformers.py [hunk 1]
```diff
@@ -111,3 +111,36 @@ def problem_save(current_event, caliper_event):
         'ip': current_event['ip']
     })
     return caliper_event
+
+
+def save_problem_success(current_event, caliper_event):
+    """
+        The server emits save_problem_success events when a problem is saved successfully.
```
Comment (asamolion) [inline openedx/features/caliper_tracking/transformers/problem_transformers.py]: Please fix the whitespace in the docstring.

Comment (mumarkhan999) [reply openedx/features/caliper_tracking/transformers/problem_transformers.py]: done in the latest commit.

**Pull request merged**